api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -269,4 +297,7 @@ message ExtProcOverrides {
 
   // Set a different gRPC service for this route than the default.
   config.core.v3.GrpcService grpc_service = 5;
+
+  // Options related to the sending and receiving of dynamic metadata
+  MetadataOptions metadata_options = 6;
```

Comment:
Since this is related to dynamic metadata, I think the name dynamic_metadata_options is preferable and more clear.

Reply:
We can ignore my comment here, as dynamic in the name is redundant, since the term metadata in xDS is used specifically for dynamic metadata